modules/social_features/social_tagging/social_tagging.install

```diff
@@ -237,3 +237,37 @@ function social_tagging_update_11201(): void {
 
   $config_storage->write('language.content_settings.taxonomy_term.social_tagging', (array) $source->read('language.content_settings.taxonomy_term.social_tagging_11201'));
 }
+
+/**
+ * Install Tagging base field to the media entity type.
```

Comment:
@ksis Could you explain why this update hook is needed?
What happens if Media module will be enabled after this hook update runs?

Reply:
@ribel This hook creates a column in the database. If the media module will be installed after this hook, the field will be added in social_tagging_modules_installed - https://github.com/goalgorilla/open_social/pull/2982/files#diff-19de5cecbacf0e4c7d03f7e5f50073625b76b77849e19185b222a591a96e03dfR665

Reply:
@ksis Ok, I see, thanks!